packages/good-design/src/stories/apps/magik/Magik.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import MagikWidget from "@magiklabs/react-sdk";
+
+export const MagikWidgetOfframp = {
+  args: {
+    isProdUrl: true,
+    defaultCountry: "NG",
+    enableDebugLogs: false,
+    environment: "staging",
+    partnerContext: {},
+    onClose: () => {},
+    apiKey: "r6eMBtUkT2A_aStNmMAfpaOT8fFs0Bew",
+    tokens: [{
+      address: '0x62b8b11039fcfe5ab0c56e502b1c372a3d2a9c7a',
+      chainId: 'celo',
+    }],
+    fiatAmount: "30",
```

Comment:
what is fiatamount?

Reply:
This is the fiat amount (in the currently selected country's currency) that will be prepopulated as the default on / offramp amount when the widget loads.

For example: if country is `IN`, and default amount is `3000`, the initial quote request will go for `3000 INR`.

However, if it is above or below the min / max limit, we will resend the request with the closest amount within the range limit.

So you don't need to worry about this - most of the time, it will just send the min amount possible as the default.